doc/source/jupyterhub/customizing/user-environment.md

```diff
@@ -69,6 +69,19 @@ If you'd like users to select an environment from **multiple docker images**,
 see {ref}`multiple-profiles`.
 ```
 
+The Docker image will be started with the [`ENTRYPOINT`](https://docs.docker.com/engine/reference/builder/#entrypoint) and [`CMD`](https://docs.docker.com/engine/reference/builder/#cmd) defined in the image.
+You can override `CMD`, by setting `singleuser.cmd`.
+
+```{note}
+In versions of Z2JH prior to 1.0.0 the images default was overridden.
+To restore this behaviour:
+```
+
+```yaml
+singleuser:
+  cmd: jupyterhub-singleuser
+```
+
```

Comment:
I could do with some formatting help here. At the moment this renders as

![image](https://user-images.githubusercontent.com/1644105/116823573-2aaa5600-ab7d-11eb-89be-0d1e92d4f633.png)

If I try to put the yaml snippet inside the note:
``````
```{note}
In versions of Z2JH prior to 1.0.0 the images default was overridden.
To restore this behaviour:

   ```yaml
   singleuser:
     cmd: jupyterhub-singleuser
   ```

```

``````

It looks like this
![image](https://user-images.githubusercontent.com/1644105/116823548-10707800-ab7d-11eb-8ba5-ff5d8173862c.png)

Reply:
I think perhaps you can use four backticks on the note directive, try that!

Reply:
That did it, thanks!